src/kOS/Function/BuildList.cs

```diff
@@ -40,7 +40,7 @@ public override void Execute(SharedObjects shared)
                     list = shared.Vessel.PartList(listType, shared);
                     break;
                 case "files":
-                    list = ListValue.CreateList(shared.VolumeMgr.CurrentVolume.GetFileList());
+                    list = ListValue.CreateList(shared.VolumeMgr.CurrentVolume.FileList.ToList());
```

Comment:
Confirmed, `Values` works.

I found it because I specifically wanted to test to make sure the old work around still works.  So I'm running it on old scripts without implementing the new functionality first.